File: C++/3005. Count Elements With Maximum Frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& n) {
        map<int,int>m;
        for(int i:n){
            m[i]++;
        }
        int max=0,c=0;
        for(auto i:m){
            if(i.second>max)
            max=i.second;
        }
        for(auto i:m){
            if(max==i.second)
            c++;
        }
        return max*c;
    }
};

File: JAVA/3005. Count Elements With Maximum Frequency.java
class Solution {
    public int maxFrequencyElements(int[] n) {
        Map<Integer,Integer>m=new HashMap<>();
        for(int i:n)
        m.put(i,m.getOrDefault(i,0)+1);
        int max=0,c=0;
        for(int i:m.values()){
            if(i>max)
            max=i;
        }
        for(int i:m.values()){
            if(i==max)
            c++;
        }
        return max*c;
    }
}